device_doctor/test/src/fake_ios_device.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
```

Comment:
Why a fake rather than a mock?

Reply:
This is used to test function `discoverDevices`, which calls another function `deviceListOutput`.
If we mock `discoverDevices` directly, we will lose testing logic of this function.
Instead if we use fake, `deviceListOut` can be manipulated so that we can test the logic of `discoverDevices`. This is how we do here.